README.md

```diff
@@ -139,7 +139,7 @@ const OrbitDB = require('orbit-db')
 
 const ipfs = IpfsApi('localhost', '5001')
 const orbitdb = new OrbitDB(ipfs)
-const db = await orbitdb.log('hello')
+const db = orbitdb.log('hello')
```

Comment:
`orbitdb.log` is an async function so it will return a promise here. Maybe `orbitdb.log('hello').then(db => // do stuff with db )` ?